src/ontology/ro-edit.owl

```diff
@@ -17,38 +18,35 @@ Prefix(oboInOwl:=<http://www.geneontology.org/formats/oboInOwl#>)
 
 Ontology(<http://purl.obolibrary.org/obo/ro.owl>
 <http://purl.obolibrary.org/obo/ro-edit.owl>
-Import(<http://purl.obolibrary.org/obo/ro/components/core.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/components/el-constraints.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/components/generated-axioms.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/components/rohom.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/components/temporal-intervals.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/cl_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/envo_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/go_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/omo_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/orcidio_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/other_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/pato_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/uberon_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/obi_import.owl>)
-Annotation(dc:description "The OBO Relations Ontology (RO) is a collection of OWL relations (ObjectProperties) intended for use across a wide variety of biological ontologies."@en)
+Import(<http://purl.obolibrary.org/obo/ro/annotations.owl>)
```

Comment:
It would be important to resolve all of these spurious changes before finishing the PR

Reply:
I'll update the branch from master.

Reply:
I have no idea why that happened

Reply:
This branch started before we changed the repository to use the ODK structure.